public/app/core/components/AppChrome/AppChromeService.tsx

```diff
@@ -35,14 +35,14 @@ export class AppChromeService {
     layout: PageLayoutType.Canvas,
   });
 
-  public setMatchedRoute(route: RouteDescriptor) {
+  setMatchedRoute(route: RouteDescriptor) {
```

Comment:
Are all the typescript changes unrelated?

Reply:
They are not. I must've run a linter somewhere. I'll revert this.

Reply:
Fixed.